### PR TITLE
feat(i18n): add swedish language

### PR DESCRIPTION
**Pull request by antonmoller**
Add Swedish for @valibot/i18n

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

**Comment by antonmoller** [comment]
Yeah, @Livolofsson should be able to review it. I'll ask her

**Comment by fabian-hiller** [comment]
I am happy to merge this soon after it is reviewed by another native speaker.

File: packages/i18n/src/sv.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    creditCard:     (issue) => `Ogiltigt kreditkort: Fick ${issue.received}`,
+    cuid2:          (issue) => `Ogiltigt Cuid2: Fick ${issue.received}`,
+    custom:         (issue) => `Ogiltig indata: Fick ${issue.received}`,
+    customAsync:    (issue) => `Ogiltig indata: Fick ${issue.received}`,
+    decimal:        (issue) => `Ogiltig decimal: Fick ${issue.received}`,
+    email:          (issue) => `Ogiltig email: Fick ${issue.received}`,
+    emoji:          (issue) => `Ogiltig emoji: Fick ${issue.received}`,
+    endsWith:       (issue) => `Ogiltigt slut: Förväntade ${issue.expected}, men fick ${issue.received}`,
+    excludes:       (issue) => `Ogiltigt innehåll: Förväntade ${issue.expected}, men fick ${issue.received}`,
+    finite:         (issue) => `Ogiltig ändlig: Förväntade ${issue.received}`, // todo
```
Comment (Livolofsson) [inline packages/i18n/src/sv.ts:19]: Remove forgotten todo

File: packages/i18n/src/sv.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    hash:           (issue) => `Ogiltig hash: Fick ${issue.received}`,
+    hexColor:       (issue) => `Ogiltig hexadecimal färg: Fick ${issue.received}`,
+    hexadecimal:    (issue) => `Ogiltig hexadecimal: Fick ${issue.received}`,
+    imei:           (issue) => `Ogiltigt IMEI: Fick ${issue.received}`,
+    includes:       (issue) => `Ogiltigt innehåll: Förvätade ${issue.expected}, men fick ${issue.received}`,
```
Comment (Livolofsson) [inline packages/i18n/src/sv.ts:24]:
```suggestion
    includes:       (issue) => `Ogiltigt innehåll: Förväntade ${issue.expected}, men fick ${issue.received}`,
```

File: packages/i18n/src/sv.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    integer:        (issue) => `Ogiltigt heltal: Fick ${issue.received}`,
+    ip:             (issue) => `Ogiltig IP: Fick ${issue.received}`,
+    ipv4:           (issue) => `Ogiltig IPv4: Fick ${issue.received}`,
+    ipv6:           (issue) => `Ogiltig IPv6: Fick ${issue.received}`,
+    isoDate:        (issue) => `Ogiltigt datum: Fick ${issue.received}`,
+    isoDateTime:    (issue) => `Ogiltig datum-tid: Fick ${issue.received}`, 
+    isoTime:        (issue) => `Ogiltig tid: Fick ${issue.received}`,
+    isoTimeSecond:  (issue) => `Ogiltig sekund: Received ${issue.received}`,
```
Comment (Livolofsson) [inline packages/i18n/src/sv.ts:32]:
```suggestion
    isoTimeSecond:  (issue) => `Ogiltig sekund: Fick ${issue.received}`,
```

File: packages/i18n/src/sv.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    isoTimestamp:   (issue) => `Ogiltig tidsstämpel: Fick ${issue.received}`,
+    isoWeek:        (issue) => `Ogiltig vecka: Fick ${issue.received}`,
+    length:         (issue) => `Ogiltig längd: Förväntade ${issue.expected}, men fick ${issue.received}`,
+    mac:            (issue) => `Ogiltig MAC: Fick ${issue.received}`,
+    mac48:          (issue) => `Ogiltig 48-bit MAC: Fick ${issue.received}`,
+    mac64:          (issue) => `Ogiltig 64-bit MAC: Fick ${issue.received}`,
+    maxBytes:       (issue) => `Ogiltiga bytes: Fick ${issue.expected}, men förväntade ${issue.received}`,
+    maxLength:      (issue) => `Ogiltig längd: Förväntade ${issue.expected}, men fick ${issue.received}`,
+    maxSize:        (issue) => `Ogiltig storlek: Förväntade ${issue.expected}, men fick ${issue.received}`,
+    maxValue:       (issue) => `Ogiltig värde: Förväntade ${issue.expected}, men fick ${issue.received}`,
```
Comment (Livolofsson) [inline packages/i18n/src/sv.ts:42]:
```suggestion
    maxValue:       (issue) => `Ogiltigt värde: Förväntade ${issue.expected}, men fick ${issue.received}`,
```

**Review by Livolofsson — COMMENTED** [review]
Just some tiny spelling mistakes, should be good to go when they're fixed!

**Comment by fabian-hiller** [comment]
Thanks! I will review and merge this soon!

**Pull request merged**